corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -200,6 +201,7 @@ <h3 data-bind="text: $root.activeCaseType()" style="display: inline-block;"></h3
           <div class="table-row table-header">
           <div class="row-item-small"></div>
           <div class="row-item">{% trans "Case Property" %}</div>
+          <div class="row-item">{% trans "Label" %}</div>
```

Comment:
You might just have to add an extra column for the case property group rows. Looking at the before/after screenshots, the "Label" heading is now stretching over the group name input a little bit and the "Case Property Group" text on the group row is a little off from the "Data Type" heading.

Reply:
cbaecc7c21e